src/math/math.cr

```diff
@@ -3,7 +3,8 @@ require "./libm"
 module Math
   extend self
 
-  PI    = 3.14159265358979323846
+  PI    =  3.14159265358979323846
+  TAU   = 6.283185307179586476925
```

Comment:
```suggestion
  TAU   = PI * 2
```

Reply:
Please no. I'm not sure the compiler will be able to inline it like that.

Reply:
I was expecting this, and that's what I thought initially. That mathematically correct, but with how computers work, it does not produce a correct result.
```cr
p Math::PI * 2 #=> 6.283185307179586
```
`6.283185307179586476925` != `6.283185307179586`.
Similarly, `Math::TAU / 2` != `Math::PI`.

I have checked how this constant in implemented in C#, it is done like this.

Reply:
Technically, the result may be correct, because it is a rounded number, but at the end is we lose precision.

Reply:
Technically, the result may be correct, because it is a rounded number, but the point is at the end, precision is lost.

Reply:
In any case, the obviously only correct solution would be `PI = TAU / 2` 😄